01_fib/hsy3418/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+	first := 1
+	second := 1
+	for i := 0; i < n; i++ {
```

Comment:
I would suggest that it makes a little more sense for this to count from 1 to n, rather than 0 to n-1 - i.e. `for i := 1; i <= n; i++`, because then for each iteration of the loop, `i` is that fibonacci number - i.e. i == 1 is the first fib, i == 2 is the second fib. It makes it a little easier to think about.

Reply:
Done